packages/instantsearch.js/src/lib/__tests__/InstantSearch-test.tsx

```diff
@@ -1359,10 +1363,15 @@ describe('scheduleStalledRender', () => {
     search.mainHelper!.search();
     search.mainHelper!.search();
 
+    await wait(0);
+
+    // search starts
```

Comment:
It's more asserting "has been called another time since the last assert (they are 1,2,3 in order). Do you know another way of describing that?

Reply:
Ah yeah, it always gets me because it's mutable—I usually clear mocks in between to add clarity.

One way we can clarify is maybe this:

```suggestion
    // search starts, `widget.render` is immediately called once more
```

Reply:
Resetting mock makes sense, then I can just do hasBeenCalledTimes(1) multiple times